python-threatexchange/threatexchange/cli/match_cmd.py

```diff
@@ -225,23 +315,48 @@ def _match_file(
     return index.query(s_type.hash_from_file(path))
 
 
+def _group_hashes_by_prefix(
+    path: pathlib.Path,
+    settings: CLISettings,
```

Comment:
nit: unused - also, I prefer to avoid passing this around because it is somewhat of a god object